sqltest/generic.go

```diff
@@ -73,8 +73,19 @@ func mustTruncateTables(t testing.TB, tr Truncator, tables ...string) {
 func truncateTables(t testing.TB, tr Truncator, agent Agent, tables ...string) error {
 	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
 	defer cancel()
+	ch := make(chan error)
```

Comment:
Is `ch` a common idiom? I'd think `errC` would be what to use. Just a nitpick, I'm happy to be wrong.

Reply:
I think `errC`, or `errc` helps distinguish an error channel from a results channel. Since I only have an error channel, because it is the result, I figured there was no need. Also since the whole deal is a handful of lines, I thought this would be enough. I'm indifferent, though.